pkg/io/storage/postgre.go

```diff
@@ -0,0 +1,90 @@
+package storage
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+type postgreVault struct {
+	db *sqlx.DB
+}
+
+// NewPostgreVault create new postgreSQL  client
+func NewPostgreVault(p *sqlx.DB) *postgreVault {
+	pv := &postgreVault{
+		db: p,
+	}
+	return pv
+}
+
+// SaveData put data in postgres storage by key and encoded value
+// 	key to set in postgres storage
+// 	encoded value to storage
+func (r *postgreVault) SaveData(key, encodedValue []byte) error {
+	ctx := context.Background()
+	if bytes.Equal(key, []byte("")) {
+		return errors.New("postgres: key can't be nil ")
+	}
+	tx, err := r.db.BeginTxx(ctx, nil)
+	if err != nil {
+		return fmt.Errorf("postgres: can't begin transaction  %w", err)
```

Comment:
Redundant spaces and missing `:`
```suggestion
		return fmt.Errorf("postgres: can't begin transaction: %w", err)
```